app/components/content/Screenshot.vue

```
@@ -0,0 +1,65 @@
+  src: string
+  alt?: string
+  href?: string
+}>()
+</script>
+
+<template>
+  <figure class="screenshot">
+    <div class="screenshot__frame">
+      <component
+        :is="href ? 'a' : 'div'"
+        :href="href"
```

ESLint

[error] Expected same-name shorthand.

(vue/v-bind-style)